model/user/user.go

```diff
@@ -159,3 +174,12 @@ func (u *DBUser) IncPatchNumber() (int, error) {
 	return dbUser.PatchNumber, nil
 
 }
+
+func IsValidSubscriptionPreference(in string) bool {
+	switch in {
+	case event.EmailSubscriberType, event.SlackSubscriberType:
```

Comment:
nit, `case in PreferenceEmail, PreferenceSlack:`, since you've redefined them locally?

Reply:
They're different types. If I have to do that I have to type assert to string, which is messier